app/views/draws/_form.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= simple_form_for @draw do |f| %>
   <%= f.input :name %>
   <%= f.input :intent_deadline %>
+  <%= f.input :lock_intent %>
```

Comment:
Do we want to make this a button near the intent report instead? We don't even have a way for admins to navigate to `/draws/:id/edit` at the moment.